<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QDialog" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>170</y>
     <width>321</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="gestionBagage">
      <property name="text">
       <string>GESTION BAGAGE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="gestionPasager">
      <property name="text">
       <string>GESTION PASSAGER</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="gestionVol">
      <property name="text">
       <string>GESTION VOL</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="gestionBillet">
      <property name="text">
       <string>GESTION BILLET</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="gestionEmployee">
      <property name="text">
       <string>GESTION EMPLOYEE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="gestionAvion">
      <property name="text">
       <string>GESTION AVION</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="logout">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>LOGOUT</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
